src/components/articles-cards/index.less

```diff
@@ -0,0 +1,93 @@
+
+
+.articles-cards-list--vertical() {
+  .category-items {
+    display: flex;
+    flex-wrap: wrap;
+  }
+  .item-featured-media {
+    width: 100%;
+    float: none;
+    clear: both;
+    max-width: 100%;
+  }
+  .category-item-title,
+  .category-item-header {
+    margin-bottom: 0;
+  }
+}
+
+.articles-cards-list {
+  .category-title {
+    font-weight: 700;
+    text-transform: uppercase;
+    letter-spacing: normal;
+    font-size: 16px;
+    a {
+      color: #ff0036;
+    }
+  }
+  .category-items {
+    clear: both;
+    list-style: none;
+    padding: 0;
+  }
+  .category-item {
+    margin-bottom: 30px;
+    article {
+      overflow: hidden;
+    }
+  }
+  .item-featured-media {
+    float: left;
+    width: 50%;
+    max-width: 364px;
+    margin: 0 20px 0 0;
```

Comment:
Please, add the empty lines between the style definitions

Reply:
I don't want to take care of empty line. Will log an issue to integrate some LESS linter